bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/internal/findandreplace/IFindReplaceLogic.java

```diff
@@ -68,7 +68,7 @@ public interface IFindReplaceLogic {
 
 	/**
 	 * {@return whether incremental search may be performed by the
-	 * find/replace-logic based on the currently active options}
+	 * find/replace-logic based on the currently active options.}
```

Comment:
Please revert this unrelated change

Reply:
Addressed in https://github.com/eclipse-platform/eclipse.platform.ui/commit/d2b76269413f3578e8a605d2410049c3801569be